src/EnergyPlus/HVACUnitaryBypassVAV.cc

```diff
@@ -1511,6 +1520,9 @@ namespace HVACUnitaryBypassVAV {
             //   Set UnitarySys flag to FALSE and let the heating coil autosize independently of the cooling coil
             DataAirLoop::AirLoopControlInfo(AirLoopNum).UnitarySys = false;
             DataAirLoop::AirLoopControlInfo(AirLoopNum).FanOpMode = CBVAV(CBVAVNum).OpMode;
+            // check for set point manager on outlet node of CBVAV
+            CBVAV(CBVAVNum).OutNodeSPMIndex =
+                SetPointManager::getSPMBasedOnNode(OutNode, SetPointManager::iCtrlVarType_Temp, SetPointManager::iSPMType_MixedAir, SetPointManager::CtrlNodeType::reference);
```

Comment:
grab the specific SPM index for a mixed air SPM if connected to the CBVAV outlet node.